demo/controllers/dialogEditorController.ts

```diff
@@ -9,7 +9,7 @@ export default class DialogEditorController {
           'label': 'New tab',
```

Comment:
@dtaylor113 can you change this to "New Tab" ( capital T ) ?

Reply:
Ok, fixed